modules/ROOT/pages/migration/index.adoc

```diff
@@ -1,729 +1,1008 @@
+[cols="1,1"]
+|===
+|Before | Now
 
+a|
 [source, javascript, indent=0]
 ----
-const schema = makeAugmentedSchema({
-  typeDefs,
-  config: {
-    query: false, // exclude all queries
-    mutation: false // exclude all mutations
-  }
-});
+const neoSchema = new Neo4jGraphQL({
+    typeDefs,
+    config: {
+        driverConfig: {
+            database: "different-db"
+        },
+    },
+})
 ----
-
-Or to exclude queries for a specific type like so:
-
+a|
 [source, javascript, indent=0]
 ----
-const schema = makeAugmentedSchema({
-  typeDefs,
-  config: {
-    query: {
-      exclude: ["NameOfTypeToExclude"] // exclude query for type NameOfTypeToExclude
-    },
-    mutation: false // exclude all mutations
-  }
-});
-----
+import { ApolloServer } from '@apollo/server';
+import { startStandaloneServer } from '@apollo/server/standalone';
+import { Neo4jGraphQL } from "@neo4j/graphql";
+import neo4j from "neo4j-driver";
 
-To achieve the same behaviour with the new `@neo4j/graphql` library, the GraphQL schema has to be extended instead. Note, no additional configuration or parameters need to be passed to the `getSchema()` function.
+const typeDefs = `#graphql
+    type User {
+        name: String
+    }
+`;
 
-To exclude all mutations for a specific type:
-[source, graphql, indent=0]
-----
-type NameOfType @exclude(operations: [CREATE, UPDATE, DELETE]) {
-  name: String
-}
-----
+const driver = neo4j.driver(
+    "bolt://localhost:7687",
+    neo4j.auth.basic("neo4j", "password")
+);
 
-Or to exclude all queries and all mutations for a specific type like so:
+const neoSchema = new Neo4jGraphQL({ typeDefs, driver });
 
-[source, graphql, indent=0]
-----
-type NameOfTypeToExclude @exclude {
-  name: String
-}
-----
-For more information regarding the above used `@exclude` directive, see xref::/schema-configuration/type-configuration.adoc#_exclude_deprecated[`@exclude`]
+const server = new ApolloServer({
+    schema: await neoSchema.getSchema(),
+});
 
-=== Database Configuration
+await startStandaloneServer(server, {
+    context: async ({ req }) => ({ sessionConfig: { database: "my-database" }}),
+});
+----
+|===
 
-==== Driver
+=== `config.enableRegex`
 
-Once a schema has been generated, it can be passed into a GraphQL server instance with the driver in the context, identical to using the old library. For example, using Apollo Server and using the schema generated above:
+This has been replaced by `MATCHES` in features.filters. 
+With this change comes more granularity in the feature configuration. 
+You can now enable the `MATCHES` filter on `String` and `ID` fields separately:
 
 [source, javascript, indent=0]
 ----
-const { ApolloServer } = require("apollo-server");
-
-const driver = require("./driver");
-const schema = require("./schema");
-
-const server = new ApolloServer({ schema, context: { driver } });
-
-server.listen().then(({ url }) => {
-    console.log(`@neo4j/graphql API ready at ${url}`);
+neoSchema = new Neo4jGraphQL({
```

Comment:
Up to you if it's sensible to keep the before/after format, in case you want the previous way was:

```typescript
neoSchema = new Neo4jGraphQL({
    typeDefs,
    config: {
        enableRegex: true
   }
});
```